src/classes/content/wordpress/class-abstract-wordpress-content-service.php

```diff
@@ -99,32 +104,24 @@ public function set_about_jsonld( $content_id, $value ) {
 		 * as `null` if we directly pass it to the prepare function(). So its necessary
 		 * to make the query conditional based on the $value
 		 */
-		if ( null === $value ) {
-			return $wpdb->query(
-				$wpdb->prepare(
-					"
-			UPDATE {$wpdb->prefix}wl_entities
-			SET about_jsonld = NULL
-			WHERE content_id = %d AND content_type = %d
-			",
-					$content_id->get_id(),
-					$content_id->get_type()
-				)
-			);
+		$query = "UPDATE {$wpdb->prefix}wl_entities SET ";
+		$query .= ( is_null( $value ) ) ? "about_jsonld = NULL, " : "about_jsonld = %s, ";
+		$query .= ( is_null( $match_name ) ) ? "match_name = NULL " : "match_name = %s ";
+		$query .= "WHERE content_id = %d AND content_type = %d";
+
+		$params = array();
+
+		if ( ! is_null( $value ) ) {
```

Comment:
This is quite unreadable, the ifs are scattered around between the query above and the params here so that in the future it'll break if I change the query and skip this part.

Can you put them together?